rules/game_process.tex

```diff
@@ -33,9 +33,15 @@ \subsection{Robot States}
     So while in the \texttt{unstiff} state the robot is not allowed to move in any fashion! After booting, the robots are in their \texttt{unstiff} state.
     Pressing the chest button once while in the \texttt{unstiff} state, permits the robot to stiffen its joints and return to the \texttt{initial} state, or a state as indicated by GameController.
 
-  \item[Initial.] The robots are not allowed to be moving in any fashion besides initially standing up.
+    \item[Initial.] The robots are free to move at teams convenience and humans are allowed to interact with the robots. 
+    This state is not limited in time and teams have access to the field.
+    The GameController will activate this state before \texttt{visual} (i.e at the beginning of a half and during a timeout).
     Shortly pressing the chest button will switch the robot to the \texttt{penalized} state.
 
+    \item[Visual.] The robots are not allowed to move in any fashion besides initially standing up and moving their heads.
+    In \texttt{visual}, the robots are awaiting a visual indication from the referee which signals the transition to \texttt{ready}. 
+    This state is used for Champions Cup matches only. Reference to \texttt{visual} in a Challenge Shield matches refer to \texttt{ready}. 
```

Comment:
I would rephrase the second sentence to be more like: 

This state is used for Champions Cup matches only. For Challenge Shield matches, this state is skipped, and it is transitioned directly to \texttt{ready}.